AccountingSystem/private/Plotters/BaseReporter.py

```diff
@@ -73,23 +77,23 @@ class BaseReporter(DBUtils):
            1)),
       'jobs': (
           ('jobs / hour',
-           1 / 3600.0,
+           1 // 3600.0,
```

Comment:
This is a change. Before it was 
```
>>> 1 / 3600.0
0.0002777777777777778
```
now it is
```
>>> from __future__ import division 
>>> 1 // 3600.0
0.0
```
is this really intentional?

Reply:
This is what really scares me with the migration. I am surprised that we have no test coverage for that specific code though.
I am more and more thinking that we should have a release with ONLY modernization, and a dedicated hackaton. Otherwise, everytime there is such a PR, we will have doubts whether a new problem comes from that or from a new "feature"

Reply:
I would strongly support that!... I was also having a look at this PR that was scaring me...
and this one spotted by Marko is a real bug introduced IMHO...

Reply:
Yes this is a real bug introduced, good catch Marko. 
As I wrote above, the new divisions are tricky. I started looking at them in the PR mostly to get experience, but I didn't change them all (in fact, you may notice I only reached until letter "C" because I was scared of going further, and wanted a second eye, hence this PR). The point is exactly that we don't have unit tests for the part of the code that is older, and where most of the modernization changes go.

I think we should introduce these changes little by little (like we do with autopep8...), not in a single PR or release, because:
- we need to all get experience with them, it should not be a one-person business
- it would be easier to spot bugs (like this one) if the PRs are small (and this one is already too big).
- ultimately, the changes are just too many. Within the trial I made with PR #4145 I realized that it's a long way to go.

I would split these modernization/python 3 changes into 2 parts:
1) those that don't introduce serialization issues, that could go slowly in
2) those that do, and that would need to wait for JSON serialization

Reply:
I agree that it should go little by little and that we should share the work and the experience. But we could still do that in one release, and distribute the work among us.

Reply:
It might be better to leave division changes to a dedicated PR (or several) as they need to be checked much more closely that the whitespace changes.

I expect there will be very few cases were integer division is actually being used intentionally. Could we try running with `python -Qwarn` and collect the logs? Once there are no warnings we can be reasonably sure it's safe to put the future import in every file (or use `-Qnew`).

Reply:
This is a good suggestion. I will consider it.